Docs/README.md

```diff
@@ -54,15 +54,15 @@ Docker image is by pushing back the compiled code into the GitHub repository
 then the Docker public repository can build automatically by linking and
 configuring specific build trigger.
 
-### Octo-App (Octobot Application System)
+### Octo-App (OCTOBOT Application System)
 
 Application preparation sub-component is required to install and configure the
 requirement as defined in the main repository such as package installation
 (Python Kubernetes API, UI bootstrap method ...), basic environment
-configuration (Python/BASH/NPM environment ...) and others. User/application
+configuration (Python/BASH/NPM environment ...) and others. Third-party user/application
```

Comment:
I think `user` will be easier to understand.

Reply:
We have several users. Third-party user is considered as "operator" or "application".